test/envtest/dlbdeviceplugin_controller_test.go

```diff
@@ -102,4 +105,49 @@ var _ = Describe("DlbDevicePlugin Controller", func() {
 			}, timeout, interval).ShouldNot(Succeed())
 		})
 	})
+	It("upgrades", func() {
+		client := k8sClient
+		ctx := context.TODO()
+		ns := metav1.NamespaceSystem
+		name := "dlbdeviceplugin-test"
+		image0 := "intel/intel-dlb-plugin:0.22.0"
+		image := "intel/intel-dlb-plugin:" + controllers.ImageMinVersion
+		controllers.UpgradePlugins = true
+		controllers.EnvtestMode = true
+
+		ds0 := deployments.DLBPluginDaemonSet()
+
+		ds0.ObjectMeta.Namespace = ns
+		ds0.Spec.Template.Spec.Containers[0].Image = image0
+
+		Expect(client.Create(ctx, ds0) == nil)
```

Comment:
I don't know about ginkgo that much, would it look nicer to have 'By("blah blah")'?